src/components/AlertModal/AlertModal.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FunctionComponent, useMemo } from "react";
+import { Modal, StyleSheet, Text, View } from "react-native";
+import AlertLogo from "../../../assets/icons/alert.svg";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { SecondaryButton } from "../Layout/Buttons/SecondaryButton";
+import { DangerButton } from "../Layout/Buttons/DangerButton";
+import { size } from "../../common/styles";
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    width: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+    alignSelf: "center",
+    backgroundColor: "rgba(0,0,0,0.6)"
+  },
+  modalView: {
+    marginBottom: 0,
+    paddingTop: 24,
```

Comment:
We have a set of styling utilities that define the various sizes, so in this case it should be `size(3)`

Reply:
Updated